docs/data/pages.ts

```diff
@@ -438,7 +438,8 @@ const pages: MuiPage[] = [
               { pathname: '/x/react-charts/legend' },
               { pathname: '/x/react-charts/stacking' },
               { pathname: '/x/react-charts/styling' },
-              { pathname: '/x/react-charts/tooltip', title: 'Tooltips and highlights' },
+              { pathname: '/x/react-charts/tooltip' },
+              { pathname: '/x/react-charts/highlighting' },
               {
                 pathname: '/x/react-charts/zoom-and-pan',
                 title: 'Zooming and panning',
```

Comment:
Offtopic, but I feel that this is not ideal. 🙈 
cc @samuelsycamore 
<img width="307" alt="Screenshot 2024-10-08 at 16 06 46" src="https://github.com/user-attachments/assets/ff7b641d-1769-401f-a7bc-d47cca154cf4">

Reply:
`Zooming & panning` ? :angel:

Reply:
If we remove the `preview` it will usually render in a single line

Reply:
I think it was like that until Sam renamed it as `&` should be avoided. 🙈 
Probably `Zoom and Pan` or `Zoom and panning` at this point. 🤷